adlnet/LDSS-XSS
# Search - Adds the ability to search within an LCV and create an Alias without a context

# How to Test
1. Upload the ADL language set and ZeroTrust Language sets.
2. Navigate to http://localhost:8010/uid/search to view the search page.
3. Verify that searching by Alias, Definition, and Context returns the correct values.
4. Testing Alias without context: Go to the neoterms page in the admin UI and select "Add neoterm"
5. Fill in the alias form field and fill in the definition form field with an already existing definition. Leave the context and context description fields blank. Submit the new Alias. You should see a warning that says something to the effect of "creating an alias without a context is not recommended." as well as a message saying that a new term has been created. The message about a new term being created is an error that has been handled by updating Django and Django-neomodel versions in a different branch. Ignore it.
6. In another tab navigate to http://localhost:7474/ and sign into the neo4j using `username: neo4j` and `password: password`
7. run `match(n) return n` to view all nodes.
8. Locate your NeoAlias and verify that it has a relationship with the correct NeoTerm node and doesn't have a relationship to a context.
